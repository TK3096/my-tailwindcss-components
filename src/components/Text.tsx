import React from 'react'

const Text = () => {
    return (
        <p>Hello, My Text</p>
    )
}

export default Text