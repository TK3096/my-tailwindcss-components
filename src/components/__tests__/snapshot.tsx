import { render } from '@testing-library/react'
import Text from '../Text'

it('renders homepage unchanged', () => {
  const { container } = render(<Text text='Tong' />)
  expect(container).toMatchSnapshot()
})
