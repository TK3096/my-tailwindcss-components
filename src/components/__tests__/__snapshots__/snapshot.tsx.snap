// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <p
    class="text-[#a1f542]"
  >
    Hello, 
    Tong
  </p>
</div>
`;
